app/src/org/commcare/activities/CommCareWiFiDirectActivity.java

```diff
@@ -110,6 +110,7 @@ public void onCreateSessionSafe(Bundle savedInstanceState) {
         super.onCreateSessionSafe(savedInstanceState);
 
         setContentView(R.layout.wifi_direct_main);
+        Log.d("Shivamm", "Adding lint issues in class which already have a lot of lint issues");

```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/b67bf14163879f9fb09208858b78318e65c45843/app/src/org/commcare/activities/CommCareWiFiDirectActivity.java#L113"></span>Using 'Log' instead of 'Timber'</td>
    </tr>
  </tbody>
</table>